### DISCUSSION TITLE
Dev Process speed (hot restart / watch / anything?)

### DISCUSSION THREAD

**Discussion by Disregard-Therest** [serverpod/serverpod#1326] [Q&A] [answered]
Hi,
is there any option to restart dev environment faster?

We are doing this through the full process with 
docker compose build --build-arg mode=development
docker compose up

and it takes at least 2 minutes every time, which is quiet devastating

**Comment by Isakdl** [comment, marked as answer]
You do not need to restart the docker containers during development! They are there to host the database and reddis instances that Serverpod communicates with.

You can simply run `dart bin/main.dart` in your serverpod server project to launch the server it self. Then kill the process with `ctrl + c` and run the command again to restart. Recompiling like this should at most take a few seconds.

**Comment by Disregard-Therest** [comment]
It works, thank you
